cni/network/network_windows.go

```diff
@@ -264,59 +263,55 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig, isIPv6Enabled bool) []p
 			protocol = policy.ProtocolUdp
 		}
 
+		hostIP, err := netip.ParseAddr(mapping.HostIp)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to parse hostIP")
+		}
+
 		// To support hostport policy mapping
 		// uint32 NatFlagsLocalRoutedVip = 1
-		rawPolicy, _ := json.Marshal(&hnsv2.PortMappingPolicySetting{
+		// To support hostport policy mapping for ipv6 in dualstack overlay mode
+		// uint32 NatFlagsIPv6 = 2
+		var flag hnsv2.NatFlags
```

Comment:
So earlier we were adding ipv6 policies based on `cniTypesCurr.Result` (which is from IPAM i suppose), but now we are directly reading the hostIP address type(v4 or v6) and based on that adding the flags ? 
What if the hostip is not provided ?(is that a valid case ?)